edge-modules/TwinTester/StatusCode.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace TwinTester
+{
+    enum StatusCode
```

Comment:
Not sure what the convention is for generic error codes so I went with leading 5s. Also, the analyzer has a 0 status code included in their enum, which we should consider removing? Cannot comment directly as I didn't change relevant code.